packages/devextreme/testing/tests/DevExpress.ui.widgets/multiView.tests.js

```diff
@@ -102,6 +101,14 @@ const animationCapturing = {
     }
 };
 
+const multiViewItemAttributeMap = (index) => {
```

Comment:
[code splitting]
let's move it to a module?

Reply:
[e88d35eb9fa4d0fcb171faaf710e3d6ef8dba941]